testsuite/exponential/ref/out.alt-icc.txt

```diff
@@ -0,0 +1,67 @@
+Compiled test.osl -> test.oso
```

Comment:
There are several naming conventions used for the alternative results, should they be standardized some.

Reply:
Examples of 3 differing naming conventions (assuming they are not out of date): 
[testsuite/geomath/ref/out.icc.txt]
[testsuite/exponential/ref/out.alt-icc.txt]
[testsuite/trig/ref/out.alt-icc.txt]
[testsuite/vector/ref/out-icc.txt]